package.json

```diff
@@ -8,10 +8,14 @@
     "@rails/ujs": "^6.0.0-alpha",
     "@rails/webpacker": "^4.2.2",
     "arctic_admin": "3.2.0",
+    "serialize-javascript": "^4.0.0",
     "turbolinks": "^5.2.0"
   },
   "version": "0.1.0",
   "devDependencies": {
     "webpack-dev-server": "^3.8.0"
+  },
+  "resolutions": {
+    "dot-prop": ">=5.2.0"
```

Comment:
I think this didn't took effect in the `yarn.lock` 🤔  maybe you missed to run `yarn install`?